test/Microsoft.Framework.Logging.Test/TraceSourceScopeTest.cs

```diff
@@ -6,7 +6,6 @@
 #if DNX451
```

Comment:
I noticed that too (but held off on removing it to not clutter this PR with unrelated changes).

I'll go ahead and add a commit to this PR that removes the `#if` around `using Moq;`.

I'll keep the other `#if` because the implementation of `TraceSourceScope` currently has `#if`s in its implementation around the calls to `Trace.CorrelationManager` and I haven't had a chance to investigate if those can be safely removed.